tests/lifecycle/lifehelper/lifehelper.go

```diff
@@ -207,6 +209,37 @@ func EnableMasterScheduling(scheduleable bool) error {
 	return err
 }
 
+func DefineDaemonSet(namespace string, image string, label map[string]string, name string) *v1.DaemonSet {
```

Comment:
Duplication. Please check ./utils/daemonset/daemonset.go

Reply:
In the ./utils/daemonset/daemonset.go, we have a nodeSelector spec. (line 27)
This nodeSelector will block from the pods to be instantiated on all the nodes and eventually break high availability. 
Also, there is another test case in the lifecycle suite called "pod scheduler" that verifies that the pods do not have nodeSelector/ nodeAffinity specs, so having this spec will cause other test cases to fail.

If the nodeSelector spec is redundant in the networking suite, let's remove it from ./utils/daemonset/daemonset.go, and i will use it and remove the code from lifehelper.

Reply:
Ok I see. No it's not redundant but we need to remove nodeSelector from  DefineDaemonSet function.
Then we need to add 1 more function RedefineDaemonSetWithNodeSelector to ./utils/daemonset/daemonset.go.
Next step is to add DefineDaemonSetWithNodeSelector to /tests/networking/nethelper/nethelper.go
The last step is to refactor networking test cases to use DefineDaemonSetWithNodeSelector instead of DefineDaemonSet.

Reply:
Done.